docs/content/preview/migrate/yb-voyager/postgresql.md

```diff
@@ -0,0 +1,42 @@
+
+Create a database user and provide the user with READ access to all the resources which need to be migrated. Run the following commands in a `psql` session:
+
+1. Create a new user named `ybvoyager`.
+
+   ```sql
+   CREATE USER ybvoyager PASSWORD 'password';
+   ```
+
```

Comment:
@amit-yb , Yes I'm aware of this issue . It is because of the Hugo version used for Netlify's preview. Once merged it will render correctly because our docs page uses an older version which doesn't have rendering issues.